src/components/SignUp.js

```diff
@@ -0,0 +1,149 @@
+import React, {Component} from 'react';
+import axios from "axios";
+import {toast} from "react-toastify";
+
+class SignUp extends Component {
+    constructor(props) {
+        super(props);
+        this.created=true;
+        this.state = {
+            username: '',
+            password: '',
+            repeatedPassword: ''
+        }
+    }
+
+    onSignupClick = () => {
+        const userData = {
+            username: this.state.username,
+            password: this.state.password
+        };
+        axios.post(`http://127.0.0.1:8000/api/signup/`, userData)
```

Comment:
Same here

Reply:
I use just `api/signup/` in this case now

Reply:
Same comment as above

Reply:
This place is PathList.js now